libff/algebra/fields/tests/test_fields.cpp

```diff
@@ -580,3 +582,17 @@ TEST(FieldsTest, BN128)
     test_field<Fq<bn128_pp>>();
 }
 #endif
+
+TEST(FieldsTest, BLS12_381)
+{
+    bls12_381_pp::init_public_params();
+    test_serialization<bls12_381_pp>();
+    test_field<bls12_381_Fq6>();
+    test_all_fields<bls12_381_pp>();
+    test_Fp12_2over3over2_mul_by_024<bls12_381_Fq12>();
```

Comment:
Just a question ... do they have a test for `mul_by_045` in the upstream changes?  If so it would be good to bring it over.  If not it's maybe OK for now (if the pairing is working then technically it's indirectly tested).

Reply:
Hm, they don't. In fact they also don't have a test for `mul_by_24` . I thought that this test was added in the Clermatics fork. If not, then maybe Scipr-lab changed the code later and removed the explicit test `test_Fp12_2over3over2_mul_by_024` .

Reply:
OK, that's kind of what I was expecting.  (Yes, that makes sense if the `024` one was added in our fork.).
(marking resolved)